exotica_core/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 find_package(TinyXML2 REQUIRED)
 find_package(ZeroMQ REQUIRED)
 find_package(PkgConfig REQUIRED)
-pkg_check_modules(MSGPACK msgpack)
+pkg_check_modules(MSGPACK REQUIRED msgpack)
```

Comment:
We could use the `MSGPACK_FOUND` to append the requirement to the list that is set in catkin_package.

Reply:
I think we could also disable the meshcat visualiser if msgpack is not installed and print an informative message that an optional feature requires msgpack.